ipatests/test_xmlrpc/tracker/base.py

```diff
@@ -111,26 +133,105 @@ def filter_attrs(self, keys):
             raise RuntimeError('The tracker instance has no attributes.')
         return {k: v for k, v in self.attrs.items() if k in keys}
 
+    def cli_command(self, name, *args, **options):
+        """
+        Prepare CLI command for testing
+        """
+        cmd = "ipa {}".format(to_cli(name))
+        for item in args:
+            cmd += " {}".format(item)
```

Comment:
Ah, silly me, I forgot how to python. Will implement both your suggestions, thank you!